parquet-column/src/main/java/org/apache/parquet/internal/column/columnindex/ColumnIndex.java

```diff
@@ -57,4 +57,16 @@ public interface ColumnIndex extends Visitor<PrimitiveIterator.OfInt> {
    */
   public List<ByteBuffer> getMaxValues();
 
+  /**
+   * @return the unmodifiable list of the repetition level histograms for each page concatenated together; used for
+   * converting to the related thrift object
+   */
+  public List<Long> getRepetitionLevelHistogram();
```

Comment:
The `public` is redundant, but I think you want to keep the code alignment.

Reply:
Yes, I don't want to add outliers here ...